src/main/resources/assets/gravisuit/lang/uk_ua.json

```diff
@@ -18,11 +18,10 @@
 
   "tooltip.gravisuit.wip" : "Робота в процесі виконання",
 
-
   "item_info.toolMode" : "Поточний режим інструменту: %1$s",
   "item_info.multiModes" : "Циклічне перемикання різних режимів",
   "item_info.silkMode" : "Режим шовкового дотику: %1$s",
-  "item_info.craftingGravitool" : "Примітка: За замовчуванням потрібен Гайковий ключ без втрат.",
+  "item_info.craftingGravitool" : "Примітка: За замовчуванням потрібен режим гайкового ключа без втрат.",
```

Comment:
The entry refers to an actual item in game, while provided translation refers to item's mode. This should be reversed I think.
![image](https://github.com/Trins-mods/Gravisuit-Classic-Edition/assets/25727222/74ef2bef-e55f-44ea-8d7b-5c6e51fb1a4b)